src/material/datepicker/calendar-body.ts

```diff
@@ -457,6 +461,16 @@ export class MatCalendarBody<D = any> implements OnChanges, OnDestroy, AfterView
     } else if (this.endValue === value) {
       return this._endDateLabelId;
     }
+
+    if (this.comparisonStart !== null && this.comparisonEnd !== null) {
+      if (value === this.comparisonStart && value === this.comparisonEnd) {
+        return `${this._comparisonStartLabelId}-${this._comparisonEndLabelId}`;
```

Comment:
The hyphen between the two should be a space

Reply:
Apologies! I thought I had updated after your previous note. Change is now reflected here: [44499b2](https://github.com/angular/components/pull/29910/commits/44499b2fa5b36f7940c656824d3308b73fab3100).